app/src/main/res/layout/profilecapture.xml

```diff
@@ -41,6 +42,21 @@
             android:layout_height="wrap_content"
             android:inputType="textPersonName" />
 
+        <TextView
+            android:id="@+id/tvOtp"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="20dp"
+            android:fontFamily="sans-serif-light"
+            android:text="6 digit OTP Code"/>
```

Comment:
we have a strings folder which has values, so we can avoid harcoding the strings